### DISCUSSION TITLE
Design Idea : Use methanol as fuel, instead of methane

### DISCUSSION THREAD

**Discussion by mokun** [mars-sim/mars-sim#609] [Ideas]
Currently, we assume the Solid Oxide Fuel Cell (SOFC) technology would be used to power vehicles and drones.

However, Direct Methanol Fuel Cell (DMFC) can hold a better promise for Martian settlers.

See pic below for an aspirational commercial product from a company called Blue World that depicts a methanol fuel cell hybrid system. 

![image](https://user-images.githubusercontent.com/1168584/177017499-8825e3d4-1271-46f7-ad0b-ef1bbacd79ca.png)

See pic below for a methanol hybrid fuel cell vehicle.
![image](https://user-images.githubusercontent.com/1168584/177017430-37499ae9-0e51-4f76-b6e8-822b95bd80d3.png)

A. SOFC's disadvantages : 
1. SOFC is a great for stationary or fixed application and poor choice for mobile use. 
2. SOFC uses ceramic electrolyte and electrodes that are extremely brittle. 
3. SOFC is not suitable for vehicular applications where vibrations are a common occurrence. 
4. Since there is no paved roads on Mars yet, a bumpy ride would certainly shorten the lifetime of SOFC fuel cell stacks.

B. DMFC's advantages : 
1. Methanol is a liquid fuel that can be handled, stored and distributed easily, compared to complicated logistical handling and storing compressed methane and hydrogen gas.
2. Methanol as an organic fuel can be potentially produced and converted from methane.
3. Methanol can also be extracted from agriculture products, such as sugar cane. This is a good news for Martian settlers since sugar cane is listed as a crop that can be modeled and grown in their greenhouses.

Feel free to share your thoughts and feedback and let me know what you think.

**Comment by mokun** [comment]
Regardless, whether methane or methanol is used as the fuel, we do have options in using either using fuel cell stack or simply internal combustion engine (ICE). 

Currently in mars-sim, methane is used as the primary fuel and it's coupled with the SOFC technology in vehicles such as rovers and drones. 

This [article](https://www.futurity.org/fuel-cell-methane-1910432/) discussed one recent (circa 2018) breakthrough by inventing a new fuel catalyst, as well as one other development front in SOFC, namely, lowering operating temperature. As it was suggested, SOFC only needs the operating temperature of only 500 deg C, while a typical automobile combustion engine runs at ~600 deg C. 

Another advantage of SOFC is that it requires no steam reformer (that's responsible for converting methane and water into hydrogen fuel).

**Comment by mokun** [comment]
@bevans2000 , 

How do you like adding methanol as the new fuel for vehicles ?

This [article](https://www.futurity.org/new-catalyst-may-lead-cleaner-methanol/) from 2014 mentioned about the worldwide generation of methanol at the level of 65 million MT. 

They also talked about how scientists identified a new nickel-gallium catalyst that converts hydrogen and carbon dioxide into methanol with fewer side-products than conventional catalysts.

**Reply by bevans2000** [reply]
In theory it just needs a change to the vehicle XML because all the logic should be driven from the VehicleSpec. I would try changing one Vehicle type and see what happens. There should be no code that is hard-coded to use Methane as it should be driven from the Vehcile Spec.

**Reply by mokun** [reply]
Implementation can be tracked [here](https://github.com/mars-sim/mars-sim/issues/853).
